jetstream/pre_treatment.py

```diff
@@ -93,6 +93,17 @@ def apply(self, df: DataFrame, col: str) -> DataFrame:
         return df.loc[mask, :]
 
 
+@attr.s(auto_attribs=True)
+class NormalizeOverAnalysisPeriod(PreTreatment):
+    """Normalizes the row values over a given analysis period (number of days)."""
+
+    analysis_period: int = 1
```

Comment:
Should we rename this to `analysis_period_length`? Usually `analysis_period` is based on the `AnalysisPeriod` type, so it could be confusing

Reply:
Yea, good idea. I'll make the update before merging.